src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.mapper.FruitTransactionMapper;
+import core.basesyntax.model.Transaction;
+import core.basesyntax.service.FileReader;
+import core.basesyntax.service.ReportService;
+import core.basesyntax.service.TransactionProcessor;
+import core.basesyntax.service.TransactionProcessorImpl;
+import core.basesyntax.service.impl.CsvFileReaderImpl;
+import core.basesyntax.service.impl.CsvFileWriterImpl;
+import core.basesyntax.service.impl.ReportServiceImpl;
+import core.basesyntax.strategy.FruitService;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.OperationHandlerStrategy;
+import core.basesyntax.strategy.impl.BalanceOperationHandler;
+import core.basesyntax.strategy.impl.FruitServiceImpl;
+import core.basesyntax.strategy.impl.OperationHandlerStrategyImpl;
+import core.basesyntax.strategy.impl.PurchaseOperationHandler;
+import core.basesyntax.strategy.impl.ReturnOperationHandler;
+import core.basesyntax.strategy.impl.SupplyOperationHandler;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    private static final String READ_FILE_PATH
+            = "src/main/java/resources/FruitStore_Data.csv";
+    private static final String WRITE_FILE_PATH
+            = "src/main/java/resources/FruitStore_Report.csv";
+
+    public static void main(String[] args) {
+        System.out.println("-----------------------");
+        System.out.println("-  --= Fuit shop =--  -");
+        System.out.println("-----------------------");
+        FileReader fruitReader = new CsvFileReaderImpl();
+        List<String> linesFromFile = fruitReader.readFile(READ_FILE_PATH);
+        for (String line :
+                linesFromFile) {
+            System.out.println(line);
+        }
```

Comment:
due to our task it's not necessary to print info from given file on console so you can remove this loop